pkg/arvo/sur/identity.hoon

```diff
@@ -0,0 +1,8 @@
+/-  *ring
+|%
++$  ship-identity  [=ship =life]
```

Comment:
A question now that I've been thinking about this for awhile: should the life be part of both the `ship-identity` and the `participants` in `ring-group`? In the case of signature verification, you absolutely need to have the life number in both `%ship` and `%ring` so that you verify a message against specific key(s).

Reply:
I think so, yes.

Reply:
OK, but why? The life should certainly be part of a signature, but if someone goes and does a personal breach, they're no longer the same identity? The point of a ship name is that that's the specific identity, not the ship/life pair.

Reply:
> In the case of signature verification, you absolutely need to have the life number in both %ship and %ring so that you verify a message against specific key(s).

That seems pretty important, but indeed not semantically part of "identity".

Rift (aka breach number) on the other hand, might be worth incorporating into the identity, because it could signal change of ownership.

Reply:
I agree with Fang that Rift number seems more important than life. Not sure where this is stored, though. Suggestions?

Reply:
Here's a generic structure for signed data from old %jael:

https://github.com/urbit/urbit/blob/7df058dd6fc1878d435823fa20487aeec992780f/sys/zuse.hoon#L1079-L1083